sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -57,7 +57,8 @@ optional kernel features as defined in section 5.7 of the core SYCL
 specification.  Each device supports only certain values for the `M`,
```

Comment:
@dkhaldi: question which is not related to this particular patch and does not affect its review and merge - this sentence which starts with "Each device..." is only applicable to `joint_matrix_mad`. Spec doesn't say what it means for `joint_matrix` to be unsupported. I think it should be clarified in the spec, currently it is unclear how to match information from `get_info<ext::oneapi::experimental::info::device::matrix_combinations>` and template parameters of  `joint_matrix` type to make a conclusion that some object of `joint_matrix` is unsupported by the selected device.

Reply:
E.g., if we have
`joint_matrix<sycl::sub_group, int8_t, use::a, 12, 48, layout::row_major>`,
and our CPU is SPR, should we check in all `combination`s of `matrix_combinations` that either `atype`, `btype` or `ctype` is `matrix_type::sint8`, and if yes, one of the following pair (`max_msize`, `max_ksize`), (`max_ksize`, `max_nsize`) or (`max_msize`, `max_nsize`) is >= than (12, 48)?

Reply:
The method you are suggesting is not correct because since you already know what is the use of matrix (use::a in your example), so you should stick to that use and only compare with the corresponding type (atype in this case). Check that atype is int8_t. Then, only compare with ethe corresponding sizes (max_msize, max_ksize, msize, ksize, no need to compare with nsize). Does this make sense?

Reply:
Thanks, yes, it makes sense! Should it be added to the spec somehow?

Reply:
Yes, I think that makes sense too.  If you want to clarify this, I'd suggest adding some new text to the introduction of the section named "Query Interface".